cosmology/StructureFormation.cpp

```diff
@@ -0,0 +1,109 @@
+// StructureFormation Test
+//   Usage:
+//     srun ./StructureFormation
+//                  <path> <nx> [<ny>...] <Np> <Nt> <stype>
+//                  <lbthres> --overallocate <ovfactor> --info 10
+//     path     = path to initial conditions folder containing the file Data.csv
+//     nx       = No. cell-centered points in the x-direction
+//     ny...    = No. cell-centered points in the y-, z-, ...-direction
+//     Np       = Total no. of macro-particles in the simulation. Needs to match the IC file Data.csv
+//     Nt       = Number of time steps
+//     stype    = Field solver type (FFT and CG supported)
```

Comment:
if only FFT and CG are supported solver types, then why are other solvers types allowed in GravityFieldSolver.hpp? maybe one needs to add an exception if the solver type is not either of those